src/libs/ck-libs/Makefile

```diff
@@ -20,6 +20,10 @@ $(SIMPLE_DIRS) ParFUM ParFUM-Tops $(FEM_DIRS):
 
 FEM: $(FEM_DIRS) 
 
+ifeq ($(AMPI_WITH_LIVE_VIZ),1)
```

Comment:
How is this specified? You can pass `-DAMPI_WITH_LIVE_VIZ=1` for all other uses but this would require something different.